local-cli/server/middleware/getDevToolsMiddleware.js

```diff
@@ -24,7 +25,42 @@ function getChromeAppName() {
   }
 }
 
-module.exports = function(options, isDebuggerConnected) {
+function launchChromeDevTools(options) {
+    var debuggerURL = 'http://localhost:' + options.port + '/debugger-ui';
+    console.log('Launching Dev Tools...');
+    opn(debuggerURL, {app: [getChromeAppName()]}, function(err) {
+        if (err) {
+            console.error('Google Chrome exited with error:', err);
+        }
+    });
+}
+
+function escapePath(path) {
+  return path.indexOf(' ') > -1
+    ? '"' + path + '"' // " Can escape paths with spaces in OS X, Windows, and *nix
+    : path;
+}
+
+function launchDevTools(options, isChromeConnected) {
+    // Explicit config always wins
+    var customDebugger = process.env.REACT_DEBUGGER;
+    if (customDebugger) {
+        var projects = options.projectRoots.map(escapePath).join(' ');
+        var command = customDebugger + ' ' + projects;
+        console.log('Starting custom debugger by executing: ' + command);
+        child_process.exec(command, function (error, stdout, stderr) {
+            if (error !== null) {
+                console.log('Error while starting custom debugger: ' + error);
+            }
+        });
+    } else if (!isChromeConnected()) {
+        // Dev tools are not yet open; we need to open a session
+        // launchChromeDevTools(options.port);
```

Comment:
I'd actually prefer this way (`launchChromeDevTools(options.port);`) since `launchChromeDevTools()` doesn't need any of the other options